.github/workflows/build.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         channels: conda-forge
         python-version: ${{ matrix.python-version }}
         # this is needed for caching to work properly:
-        use-only-tar-bz2: true
+        # use-only-tar-bz2: true
```

Comment:
Why not remove the line, rather than commenting it out? Conda-forge has moved completely away for tar.bz2 so this will never be useful again.

Reply:
Agreed, will remove. 

Also, I discovered that the caching seems to only work if you use miniforge like you are preferring. If it's ok so that we can see everything works, I'll go ahead and combine your suggested PRs into this one